ra/SAPHana

```diff
@@ -2216,14 +2221,55 @@ function saphana_stop_clone() {
     super_ocf_log info "FLOW $FUNCNAME ($*)"
     local rc=0
     local primary_status="x"
-    set_hana_attribute ${NODENAME} "UNDEFINED" ${ATTR_NAME_HANA_CLONE_STATE[@]}
+    set_hana_attribute "${NODENAME}" "UNDEFINED" "${ATTR_NAME_HANA_CLONE_STATE[@]}"
     super_ocf_log debug "DBG: SET UNDEFINED"
-    check_for_primary; primary_status=$?
-    if [ $primary_status -eq $HANA_STATE_SECONDARY ]; then
-        lpa_set_lpt  10 $NODENAME
+    check_for_primary; primary_status="$?"
+    if [ "$primary_status" -eq "$HANA_STATE_SECONDARY" ]; then
+        lpa_set_lpt  10 "$NODENAME"
+    fi
+    saphana_stop; rc="$?"
+    return "$rc"
+}
+
+function recover_site_attributes_from_file()
+{
+    local srHookAttributeFile="$1"
+    local site
+
+    if [ -f "$srHookAttributeFile" ]; then
+        #
+        # need to read attribute from the file
+        #
+        # TODO: PRIO0: also allow a=b instead of a = b
+        # TODO: PRIO5: also allow multiple attributes each in an own line - for now only one attribute is needed
+        read srATTR EQS srVALUE <"$srHookAttributeFile"
+        case "$srATTR" in
+            hana_${sid}_site_srHook_* )
+                #
+                # get site name from site attribute-name
+                #
+                site="${srATTR#hana_${sid}_site_srHook_}"
+                super_ocf_log info "DEC: srHookAttribute file found - recover lost SAP HA/DR event ($srATTR=$srVALUE)"
+                #
+                # DONE: PRIO0 - add check for srATTR name here
+                #
+                case "$srVALUE" in
+                SOK )
+                      super_ocf_log info "DEC: recover attributes from file $srHookAttributeFile - calling set_SRHOOK $site SOK"
+                      set_SRHOOK "$site" "SOK"
```

Comment:
The function "set_SRHOOK" is missing. It is available in the ScaleOut RA, but currently missing here.

Reply:
Update will follow later.